src/adt/print/tree.jl

```diff
@@ -57,13 +56,13 @@ function (p::Printer)(node)
     print(xs...) = Base.print(p.io, xs...)
     println(xs...) = Base.println(p.io, xs...)
     printstyled(xs...; kw...) = Base.printstyled(p.io, xs...; kw...)
+    print_child_annotation(node, child, key) = Multiline.print_child_annotation(p.io, node, child, key)
```

Comment:
why do we change the API name here?

Reply:
Actually, I need to print `child` in some current case of YaoExpr ADT. In that case, the previou API is not enough because it does not contain the information of `child`. But I just realized it is unnecesarry and I will remove it later.